src/components/DropdownSort.jsx

```diff
@@ -0,0 +1,40 @@
+import { useEffect, useRef, useState } from 'react';
+import icArrowDown from '../assets/ic_arrow_down.svg';
+import icSort from '../assets/ic_sort.svg';
+import styles from './DropdownSort.module.css';
+
+const option = { recent: "최신순", favorite: "좋아요순", }
+
+function Dropdown({ state, setState, mode }) {
+    const [isOpen, setIsOpen] = useState(false);
```

Comment:
💬 여담
reducer 를 사용해 아래처럼 간단하게도 작성 가능합니다~

```suggestion
    const [isOpen, toggleIsOpen] = useReducer((prev) => !prev, false);
```